### PR TITLE
Bump dependencies

### PR DESCRIPTION
**Pull request by rhrazdil**
Bumping dependencies of
- containernetworking
- vishvananda
- Kubernetes 1.19.1

### CODE REVIEW

**Review by phoracek — CHANGES_REQUESTED** [review]

File: go.mod [hunk 1]
```diff
@@ -5,46 +5,43 @@ require (
...
-	golang.org/x/crypto v0.0.0-20200510223506-06a226fb4e37 // indirect
-	golang.org/x/time v0.0.0-20191024005414-555d28b269f0 // indirect
-	k8s.io/apimachinery v0.18.3
+	github.com/vishvananda/netlink v1.1.0
+	k8s.io/apimachinery v0.19.1
 	k8s.io/client-go v0.0.0-00010101000000-000000000000
 	kubevirt.io/qe-tools v0.1.6
 )
 
 // Pinned to kubernetes-1.18.3
```
[the remark this answers is not in the document]
Comment (rhrazdil) [reply go.mod]: fixed

File: go.mod [hunk 2]
```diff
@@ -5,46 +5,44 @@ require (
...
+	k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.19.1
+	k8s.io/kubectl => k8s.io/kubectl v0.19.1
+	k8s.io/kubelet => k8s.io/kubelet v0.19.1
+	k8s.io/legacy-cloud-providers => k8s.io/legacy-cloud-providers v0.19.1
+	k8s.io/metrics => k8s.io/metrics v0.19.1
+	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.19.1
 )
 
-go 1.13
+go 1.14
```
Comment (phoracek) [inline go.mod:48]: I'm afraid we have to stay on Go 1.13 for a while. Could you keep it there?

**Review by phoracek — CHANGES_REQUESTED** [review]

File: go.mod (hunk 2 above)
Comment (rhrazdil) [reply go.mod]: reverted

**Pull request merged**